source/topologies.txt

```diff
@@ -0,0 +1,36 @@
+============================
+Supported Cluster Topologies
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:ref:`c2c-mongosync` supports two cluster topologies: 
+
+- :ref:`Replica sets <replication>`
+
+- :ref:`Sharded clusters <sharded-cluster>`
+
+To sync a replica set to a replica set, see the :ref:`Quick Start Guide
+<c2c-quickstart>`.
+
+To sync a replica set to a sharded cluster, see :ref:`c2c-rs-to-sharded`.
+
+To sync a sharded cluster to a sharded cluster, see :ref:`c2c-sharded-clusters`.
```

Comment:
**Suggestion**

Consider bulleting these points, or creating a table with some headers such as:

Use Case | Link| 

Or similar to display this content.

Reply:
Made bullet points!